src/types.tsx

```diff
@@ -155,7 +157,7 @@ export type MaterialTabBarProps = MaterialTabBarOptions & {
 
 export type NavigationCommonTabOptions = {
   title?: string;
-  tabBarLabel?: React.ReactNode;
+  tabBarLabel?: string;
```

Comment:
don't believe this is correct, see: https://reactnavigation.org/docs/en/bottom-tab-navigator.html#tabbarlabel

Reply:
Good point!  This is now reverted.